lib/ansible/modules/system/parted.py

```diff
@@ -564,6 +570,14 @@ def main():
             },
             'part_start': {'default': '0%', 'type': 'str'},
             'part_end': {'default': '100%', 'type': 'str'},
+            'fs_type': {
+                'default': 'ext4',
+                'choices': [
```

Comment:
I don't think we should be using choices here as it will prevent use of other types supported by parted. Also, this should remain optional, to support the existing behavior.